{{cookiecutter.folder_name}}/texmf/styles/docswitch.sty

```diff
@@ -7,25 +7,71 @@
 % aastex
 % emulateapj
 % mnras
+% lsstdescnote
 %
 % Journal names:
-% apj (aastex)
+% apj (aastex61)
 % apjl (emulateapj)
 % prl (revtex)
 % prd (revtex)
 % mnras (mnras)
+% tex (lsstdescnote)
 %
 % Alex Drlica-Wagner <kadrlica@fnal.gov>
 %
 
 \RequirePackage{etoolbox}
 
-\newcommand{\setjournal}[1]{
+\newtoggle{revtex}
+\newtoggle{aastex}
+\newtoggle{aastex61}
+\newtoggle{mnras}
+\newtoggle{emulateapj}
+\newtoggle{lsstdescnote}
+\newtoggle{default}
+
+\newcommand{\toggle@class}[1]{%
+  % Initialize the toggles
+  \togglefalse{revtex}
+  \togglefalse{aastex}
+  \togglefalse{aastex61}
+  \togglefalse{mnras}
+  \togglefalse{emulateapj}
+
+  % Journals that map to revtex
+  \ifdefstring{#1}{revtex}{\toggletrue{revtex}}{}%
+  \ifdefstring{#1}{prl}{\toggletrue{revtex}}{}%
+  \ifdefstring{#1}{prd}{\toggletrue{revtex}}{}%
+
+  % Journals that map to aastex
+  \ifdefstring{#1}{aastex}{\toggletrue{aastex}}{}%
+
+  % Journals that map to aastex61   
+  \ifdefstring{#1}{aastex61}{\toggletrue{aastex61}}{}%
+  \ifdefstring{#1}{apjl}{\toggletrue{aastex61}}{}%
+  \ifdefstring{#1}{apj}{\toggletrue{aastex61}}{}%
+
+  % Journals that map to emulateapj   
+  \ifdefstring{#1}{emulateapj}{\toggletrue{emulateapj}}{}%
+
+  % Journals that map to mnras
+  \ifdefstring{#1}{mnras}{\toggletrue{mnras}}{}%
+
+  % Journals that map to lsstdescnote
+  \ifdefstring{#1}{lsstdescnote}{\toggletrue{lsstdescnote}}{}%
+  \ifdefstring{#1}{tex}{\toggletrue{lsstdescnote}}{}%
+
+  % Set default journal
```

Comment:
I don't have an opinion about the default (we are specifying the default in the Makefile and cookiecutter as well).